CHANGELOG.md

```diff
@@ -1,73 +1,235 @@
+- Changed use case _Devices are operational in the MD LAN network but cannot access the TS Service and clock drift is unacceptable_ so that the decision to continue/discontinue the execution of a System Function while the clocks become less accurate lies with the consumer ([#31](https://github.com/IHE/DEV.SDPi/issues/31)). 
+- Open/Closed Issues sections have been revised. Sections are now generated automatically from Github issues ([#20](https://github.com/IHE/DEV.SDPi/issues/20)).
+- Created a common section for the ACM and DEC gateways such that requirement numbers are not duplicated anymore ([#53](https://github.com/IHE/DEV.SDPi/issues/53).  
+- Rephrase R1542 to make sure, that system functions not being available don't lead to unnecessary alarms ([#180] (https://github.com/IHE/DEV.SDPi/issues/180)). 
+- Moved requirements R1542 & R1543 to the SDPi-P SES / Effectiveness section from Github issues ([#182](https://github.com/IHE/DEV.SDPi/issues/182)).
+
+### Removed
+
+### Editorial Fixes
+
+- Wrong appendix numbering in references ([#189](https://github.com/IHE/DEV.SDPi/issues/189)).
+- Missing volume number in section references ([#189](https://github.com/IHE/DEV.SDPi/issues/189)).
+- Updated all references to the sdpi-fhir Github ([#162](https://github.com/IHE/DEV.SDPi/issues/162))
+- Addressed editorial issues from the IHE Publications review ([#189](https://github.com/IHE/DEV.SDPi/issues/189))
+- Wrong appendix numbering in references ([#189](https://github.com/IHE/DEV.SDPi/issues/189))).
+- All instances of "i.e." or "e.g." replaced with "i.e.," and "e.g.,", respectivelly ([#189](https://github.com/IHE/DEV.SDPi/issues/189))).
+- Normalized "profile" to "Profile" where appropriate ([#189](https://github.com/IHE/DEV.SDPi/issues/189))).
+- Normalized "section" to "Section" when it was referring to a specific clause within the document ([#189](https://github.com/IHE/DEV.SDPi/issues/189))).
+
+## [1.0.1] - 2023-04-14
+
+### Editorial Fixes
+
+- Wrong appendix numbering offset in tables to be aligned with appendix letter ([#134](https://github.com/IHE/DEV.SDPi/issues/134)).
+- Path to XML file to correctly render Figure 3:8.7.3.17-1 Physiologic Monitor Containment Tree Example ([#136](https://github.com/IHE/DEV.SDPi/issues/136)).
+- Corrected the PUML file name in TF-1:12.1.1-1 SDPi-A to render the alert reporting sequence diagram ([#135](https://github.com/IHE/DEV.SDPi/issues/135)).
+- Replaced high-lighting from "Closed Issues" section with correct list ([#139](https://github.com/IHE/DEV.SDPi/issues/139)).
+- Remove leading dashes from table and figure captions if the section number is empty ([#144](https://github.com/IHE/DEV.SDPi/issues/144)).
+- Reviewed all "Verson Note" and version-specific (i.e, 1.0) content and either reworded or removed ([#148](https://github.com/IHE/DEV.SDPi/issues/148))
+# Changelog
```

Comment:
Something went wrong here. A lot of new lines plus the changelog chapters seem to be duplicated. I suggest to revert the file and push it again. I can do that for you if you do not know how to proceed. I just have to know which entries are to be added to the existing changelog.

Reply:
Thanks David. The tooling made a big mess of something that should have been simple. Maybe you can correct it.... What I did is:
- Added to "Added" list: - Added R0702 on private coding system identification in volume 3 and updated private mdc code mapping in volume 2 gateways ([#57](https://github.com/IHE/DEV.SDPi/issues/57)).
- Added to "Editorial Fixes" list: - Moved reference content in glossary table between columns ([#166](https://github.com/IHE/DEV.SDPi/issues/166))
- Corrected some the other lines by adding a final bracket

Reply:
I take care